tests/integration/select-component-test.js

```diff
@@ -444,3 +444,24 @@ test('Can specify a custom partial with listViewPartial', function(assert) {
       'The specified listViewPartial is rendered');
   });
 });
+
+test('Displays options when content is changed to another empty array and then immediately assigned to a content array', function() {
```

Comment:
Not sure this describes well what the test want to capture. 

You want something like

```
Select handles a change in the content array properly
```